### ISSUE TITLE
[docs] Update README.md

### ISSUE THREAD

**Issue by RalphORama** [Podcast-About-List/santa#12]
### Description of the New Feature

`README.md` needs some updates to documentation. It could use:
   - [ ] Links to PAL socials
   - [ ] "About" section
   - [ ] "Contributing" section with details about workspace setup, testing, CI, and code style
   - [ ] More badges lol (AGPL, Python 3, whatever)

If I think of more stuff I'll add it to the list.

### Added Functionality

This feature changes or adds functionality to:

   - [ ] The Discord Bot
   - [ ] The Database or API
   - [ ] Other program functionality (`.env`, scripts, etc.)
   - [x] Documentation
   - [ ] Containerization
   - [ ] CI
   - [ ] Other: 

### External Dependencies

   - [x] My feature request **does not** add any external dependencies

**Comment by abdallah1097** [comment]
I'm new to discord, do you mind explaining how this feature is initiated in discord?

**Comment by abdallah1097** [comment]
I ran the bot in discord. However, other bots created by me respond but your isn't, do you mind telling me some chat commands to enable it? e.g. my info mentioned in the design book isn't working as well

`Abdallah@DESKTOP-G6QENGL MINGW64 /d/Work/GitHub Contributions/santa (documentation-edits)
$ python santabot.py
> Starting Santa Bot...
> Loading cogs...
> Loaded cog  santabot.cogs.give
> Loaded cog  santabot.cogs.grinch
> Loaded cog  santabot.cogs.invite
> Logging into Discord...
> Santa Bot is online!
`

**Comment by RalphORama** [comment]
@abdallah1097 Commands are prefixed with a mention of the bot. I.e. `@santa give [@recipient] <present name>`

**Comment by abdallah1097** [comment]
> @abdallah1097 Commands are prefixed with a mention of the bot. I.e. `@santa give [@recipient] <present name>`

Yes still the same problem. Do you mind sending me a discord server link where Santa Bot is enabled (I'm sure you tested it this way) so I can add more examples to the README.md and guide to how to use it?
